src/app/components/authenticated-v2/results/results.component.ts

```diff
@@ -83,6 +83,7 @@ export interface ResultItem {
   perk: ArmorPerkOrSlot;
   transferState: ResultItemMoveState;
   statsNoMods: number[];
+  source: number;
```

Comment:
This should be `InventoryArmorSource` instead of `number` to make sure it is always referencing the enum values.